pkg/state/state.go

```diff
@@ -2517,6 +2519,20 @@ func (st *HelmState) appendExtraDiffFlags(flags []string, opt *DiffOpts) []strin
 	return flags
 }
 
+// appendExtraSyncFlags appends extra diff flags to the given flags slice based on the provided options.
+// If opt is not nil and opt.SyncArgs is not empty, it collects the arguments from opt.SyncArgs and appends them to flags.
+// If st.HelmDefaults.SyncArgs is not nil, it joins the arguments with a space and appends them to flags.
+// The updated flags slice is returned.
+func (st *HelmState) appendExtraSyncFlags(flags []string, opt *SyncOpts) []string {
```

Comment:
maybe add some tests for this func?

Reply:
@inteon

Reply:
@yxxhero added in https://github.com/helmfile/helmfile/pull/1375/commits/2d7ceeb970e45ae9a3b3b8936ab2c78f5f624663